base/process.jl

```diff
@@ -707,9 +707,25 @@ with a code of 0). An exception is raised if the process cannot be started.
 """
 success(cmd::AbstractCmd) = success(_spawn(cmd))
 
+struct ProcessExitedError
+	procs::Vector{Process}
```

Comment:
Four spaces for indentation, no tabs

Reply:
This is what I get for not using vim.

Reply:
Looks like you got the other tabs but not this one